### PR TITLE
Steal tunerstudio files from RusEFI

### PR DESCRIPTION
**Pull request by rusefillc**
(cherry picked from commit dcba04fbd31700f5ca4ef4f12a6e220c5713c7d0)

### CODE REVIEW

**Comment by rusefillc** [comment]
between not being exciting about BT nor TS-over-CAN @mck1117 is hesitant about TS in WBO in general

File: firmware/Makefile [hunk 1]
```diff
@@ -145,6 +145,8 @@ CPPSRC = $(ALLCPPSRC) \
           pump_control.cpp \
           uart.cpp \
           interpolation.cpp \
+          util/math/crc32.c \
+          console/binary/tunerstudio_io.cpp \
```
Comment (mck1117) [inline firmware/Makefile:149]: missing some cpp files here?

Comment (rusefillc) [reply firmware/Makefile:149]: crc32.c was actually wrong name, fixed now

File: firmware/Makefile [hunk 2]
```diff
@@ -145,6 +145,8 @@ CPPSRC = $(ALLCPPSRC) \
           pump_control.cpp \
           uart.cpp \
           interpolation.cpp \
+          util/math/crc32.c \
```
[the remark this answers is not in the document]
Comment (rusefillc) [reply firmware/Makefile]: should be C, fixed

File: firmware/Makefile (hunk 1 above)
Comment (mck1117) [reply firmware/Makefile:149]: I meant `tunerstudio.cpp`

Comment (mck1117) [reply firmware/Makefile:149]: and `tunerstudio_io_serial.cpp`, I guess

Comment (rusefillc) [reply firmware/Makefile:150]: fixed

**Comment by mck1117** [comment]
looks like `tunerstudio.h` and `thread_controller.h` are also dead in the current form

**Comment by rusefillc** [comment]
two more files removed

**Pull request merged**